modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/EmailAddressResourceTest.java

```diff
@@ -125,6 +125,24 @@ protected EmailAddress testGetEmailAddress_addEmailAddress()
 			ListTypeConstants.CONTACT_EMAIL_ADDRESS);
 	}
 
+	protected EmailAddress
+			testGetOrganizationByExternalReferenceCodeEmailAddressesPage_addEmailAddress(
+				String externalReferenceCode, EmailAddress emailAddress)
+		throws Exception {
+
+		return _addEmailAddress(
+			emailAddress, _organization.getModelClassName(),
```

Comment:
please use Organization.class.getName() - check if in the same by id we have to change it too